vsts/vsts.yaml

```diff
@@ -4,7 +4,6 @@ trigger:
   batch: true
   branches:
     include:
-    - master
```

Comment:
why?

Reply:
Right now, when any change gets merged into preview branch, we trigger 2 rolling builds on our Canary pipeline; one that runs against "master" sourcecode, and another that runs against "preview" sourcecode.
Since we don't want to test the "master" sourcecode in Canary environment, this change specifically removes the "master" rolling build trigger for "preview" branch.

Reply:
Is there a way for us to have 2 independent triggers and runs? I don't want to lose the master integration testing just because we're mostly working in a preview branch right now.

Reply:
I didn't modify the yaml getting picked up from the master branch (for prod pipeline): https://github.com/Azure/azure-iot-sdk-csharp/blob/master/vsts/vsts.yaml

So this change should in no way affect the rolling builds that we have set up on West Central US, to run against `master` sourcecode.
If it does affect that, I will revert this change.